src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+let population = document.querySelectorAll('.population');
+const totalText = document.querySelector('.total-population');
+const averageText = document.querySelector('.average-population');
+
+[...population] = [...population]
```

Comment:
Incorrect assignment to a NodeList. You should assign the mapped values to a new variable.